libs/island-ui/core/src/lib/ActionCard/ActionCard.tsx

```diff
@@ -246,7 +248,9 @@ export const ActionCard: React.FC<ActionCardProps> = ({
           <Text variant="eyebrow" color="purple400">
             {eyebrow}
           </Text>
-          <Text variant="h3">{heading}</Text>
+          <Text variant={headingVariant} as="p">
```

Comment:
We probably don't want to render this heading as a p tag, because thats bad for accessibility. I think h3 headings should be rendered as h3 tags and h4 headings should be rendered as h4 tags.

Reply:
whops - fixed

Reply:
Won't that be an accessibility issue as well (a minor one sure) when you have perhaps a h1 or no heading at all and then the next heading is a h3 in the ActionCard? Would it make sense to allow the developer to use a `as` prop?

Reply:
Hmm good point -  @oddsson - what do you think?